calicoctl/commands/node/run.go

```diff
@@ -229,17 +229,23 @@ Description:
 		envs["IP6"] = ipv6
 	}
 
-	// Create a map of read only bindings.
-	vols := map[string]string{
-		logDir:            "/var/log/calico",
-		"/var/run/calico": "/var/run/calico",
-		"/lib/modules":    "/lib/modules",
+	// Create a struct for volumes to mount.
+	type vol struct {
+		hostPath      string
+		containerPath string
 	}
 
+	// vols is a slice of read only volume bindings.
+	var vols []vol
+
+	vols = append(vols, vol{hostPath: logDir, containerPath: "/var/log/calico"},
```

Comment:
Marginally easier to just vols as a slice containing these three entries.

`vols := []vol{{hostPath: logDir, containerPath: "/var/log/calico"}, {...}, {...}}`